django/core/management/commands/loaddata.py

```diff
@@ -136,83 +149,79 @@ def loaddata(self, fixture_labels):
         # If we found even one object in a fixture, we need to reset the
         # database sequences.
         if self.loaded_object_count > 0:
-            sequence_sql = connection.ops.sequence_reset_sql(no_style(), self.models)
-            if sequence_sql:
-                if self.verbosity >= 2:
-                    self.stdout.write('Resetting sequences')
-                with connection.cursor() as cursor:
-                    for line in sequence_sql:
-                        cursor.execute(line)
+            self._reset_sequences(connection)
 
         if self.verbosity >= 1:
-            if self.fixture_object_count == self.loaded_object_count:
-                self.stdout.write(
-                    "Installed %d object(s) from %d fixture(s)"
-                    % (self.loaded_object_count, self.fixture_count)
-                )
-            else:
-                self.stdout.write(
-                    "Installed %d object(s) (of %d) from %d fixture(s)"
-                    % (self.loaded_object_count, self.fixture_object_count, self.fixture_count)
-                )
+            msg = "Installed %d object(s) {}from %d fixture(s)" % (
+                self.loaded_object_count, self.fixture_count)
+            of = ""
+            if self.fixture_object_count != self.loaded_object_count:
+                of = f"(of {self.fixture_object_count}) "
```

Comment:
May be an easier change for django reviewers if you keep this block in the same format as the one you replaced

Reply:
I removed the change in a5a69cab1a081b210d53703d6c8cd69e78362a7e given that it also confused @jacobtylerwalls https://github.com/django/django/pull/13842/files#r585972060

Even if the code is okay, it confused two separate reviewers. That's not a sign of clear code.